protocol/x/clob/abci.go

```diff
@@ -120,6 +121,13 @@ func PrepareCheckState(
 	keeper *keeper.Keeper,
 	liquidatableSubaccountIds *liquidationtypes.LiquidatableSubaccountIds,
 ) {
+	defer telemetry.MeasureSince(
```

Comment:
Added latency metrics on every step of this function except two function calls:
- `GetProcessProposerMatchesEvents`
- `SetMemclobGauges`

Let me know if anyone wants me to add latency metrics to those.